ChangeLog

```diff
@@ -1,6 +1,9 @@
 CHANGES
 =======
 
+2.0.0
+
+FEATURE: Create Orion indexes neded in MongoDB
```

Comment:
typo: neded

Reply:
In addition, I'd suggest to use the issue number and the other delete service functionality, that's it:

```
FEATURE: Create Orion indexes needed in MongoDB (#145)
FEATURE: Remove Orion and STH collections upon service deletion
```

Reply:
Fixed in 857545c04e5aed69596cd42b3b6d6f2df9ccd67a